### PR TITLE
Second paragraph

### PR DESCRIPTION
**Pull request by cudjoeab**
stay snacking friends!

### CODE REVIEW

**Comment by ShaheerC** [comment]
Wow great work abi!

File: mad-libbing.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 Fighting evil by power, winning love by emotion
 Never running from a monster, she is the one named Eden Y.
 
-She will never turn her back on her ______.
-She is always there to ______.
-She is the one in whom we can ______.
+She will never turn her back on her apetite.
+She is always there to have to bite.
+She is the one in who can clear a plate.
```
Comment (fredngo) [inline mad-libbing.txt]: 👍

File: mad-libbing.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Fighting evil by power, winning love by emotion
 Never running from a monster, she is the one named Eden Y.
 
-She will never turn her back on her ______.
-She is always there to ______.
-She is the one in whom we can ______.
+She will never turn her back on her apetite.
```
Comment (fredngo) [inline mad-libbing.txt]: Can you change apetite to stomach

**Comment by fredngo** [comment]
Awesome kthxbye

**Pull request merged**